framework/test/CMakeLists.txt

```diff
@@ -2,3 +2,4 @@
 add_subdirectory(bundles)
 add_subdirectory(driver)
 add_subdirectory(gtest)
+add_subdirectory(bench)
```

Comment:
Don't know if the lack of a carriage return will affect things with this file. Sometimes it's a problem.

Reply:
seems to build fine on all platforms